libs/tasks/ui-tasks-list/src/lib/component/tasks-list.component.scss

```diff
@@ -0,0 +1,31 @@
+.task-list__header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 40px;
+}
+
+::ng-deep.task-list__header > * {
```

Comment:
don't use `::ng-deep` this is _deprecated_

Reply:
ok